src/Pages/Withdrawals.razor

```diff
@@ -772,6 +796,50 @@
         }
     }
 
+    private async Task ShowBumpfeeModal(WalletWithdrawalRequest walletWithdrawalRequest)
+    {   
+        ulong confirmations = 0;
+        if (!string.IsNullOrEmpty(walletWithdrawalRequest.TxId))
+        {
+            try
+            {
+                var nbxplorerStatus = await NBXplorerService.GetTransactionAsync(uint256.Parse(walletWithdrawalRequest.TxId));
+                confirmations = (ulong)(nbxplorerStatus?.Confirmations ?? throw new Exception("Failed to get confirmations"));
+            }
+            catch (Exception)
+            {
+                ToastService.ShowError("Error getting confirmations");
+                return;
+            }
+        }
+        
+        if (confirmations > 0)
+        {
+            ToastService.ShowError("Bumpfee can only be used for transactions with no confirmations. " +
+                                   "This transaction has already been mined.");
+            return;
+        }
+
```

Comment:
I would then not display the button at all, a button that can be clicked and the click just tells error its not worth imo

Reply:
yep, it doesn't show but just in case that a mine gets mined after loading the page, we recheck it